var/spack/repos/builtin/packages/exciting/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Exciting(MakefilePackage):
+    """
+    exciting is a full-potential all-electron density-functional-theory package
+    implementing the families of linearized augmented planewave methods. It can
+    be applied to all kinds of materials, irrespective of the atomic species in
+    volved, and also allows for exploring the physics of core electrons. A
+    particular focus are excited states within many-body perturbation theory.
+    """
+
+    homepage = "http://exciting-code.org/"
+    url      = "http://exciting.wdfiles.com/local--files/nitrogen-14/exciting.nitrogen-14.tar.gz"
+
+    version('14', sha256='a7feaffdc23881d6c0737d2f79f94d9bf073e85ea358a57196d7f7618a0a3eff')
+
+    # as-of-yet unpublished fix to version 14
+    patch('dfgather.patch', when='@14', working_dir='src/src_xs', level=0)
+    variant('mpi', default=False, description='Use MPI')
+    variant('mkl', default=False, description='Use MKL')
+    variant('omp', default=True, description='Use OpenMP')
+    variant('scalapack', default=False, description='Use ScaLAPACK')
+    depends_on('blas+lp64')
```

Comment:
I don't think virtual dependencies support variants at the moment.

Reply:
You're right!  I removed it.